app/_hub/kong-inc/pre-function/how-to/_index.md

```diff
@@ -63,13 +54,13 @@ different priority in the plugin chain.
 5. Apply our Lua code using the `pre-function` plugin using cURL file upload:
 
     ```bash
-    curl -i -X POST http://localhost:8001/services/plugin-testing/plugins \
-        -F "name=pre-function" \
-        -F "config.access[1]=@custom-auth.lua" \
-        -F "config.access[2]=kong.log.err('Hi there Access!')" \
-        -F "config.header_filter[1]=kong.log.err('Hi there Header_Filter!')" \
-        -F "config.body_filter[1]=kong.log.err('Hi there Body_Filter!')" \
-        -F "config.log[1]=kong.log.err('Hi there Log!')"
+    curl -i -X POST http://localhost:8001/services/example-service/plugins \
+      --form "name=pre-function" \
+      --form "config.access[1]=@custom-auth.lua" \
+      --form "config.access[2]=kong.log.err('Hi there Access!')" \
+      --form "config.header_filter[1]=kong.log.err('Hi there Header_Filter!')" \
+      --form "config.body_filter[1]=kong.log.err('Hi there Body_Filter!')" \
+      --form "config.log[1]=kong.log.err('Hi there Log!')"
```

Comment:
```suggestion
      --form "config.access[1]=@custom-auth.lua" \
      --form "config.access[2]=kong.log.err('Test Access')" \
      --form "config.header_filter[1]=kong.log.err('Test Header_Filter!')" \
      --form "config.body_filter[1]=kong.log.err('Test Body_Filter!')" \
      --form "config.log[1]=kong.log.err('Test Log!')"
```


The "Hi there" didn't feel right, maybe there is a better thing we can use than test here.